media/transcoder/job_test.py

```diff
@@ -252,3 +357,11 @@ def _get_job_state(capsys, job_id):
 
     out, _ = capsys.readouterr()
     assert job_succeeded_state in out
+
+
+def _get_file_in_bucket(capsys, directory_and_filename):
```

Comment:
This name is deceptive too - it should be more like `_assert_file_in_bucket`.

Reply:
Changed name to _assert_file_in_bucket